src/test/experiments.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from "chai";
+import {
+  enableExperimentsFromCliEnvVariable,
+  getSnapshotOfExperimentPreferences,
+} from "../experiments";
+
+describe("experiments", () => {
+  describe("enableExperimentsFromCliEnvVariable", () => {
+    it("should enable some experiments", () => {
+      process.env.FIREBASE_CLI_EXPERIMENTS = "experiments,not_an_experiment";
```

Comment:
This will persist into other tests after this is run. Consider creating a `before` that either stores or sets this to `""`, then in the test set it and run the function, then restore or reset to `""` in an `after`. ~Probably will have to run the `enableExperimentsFromCliEnvVariable` again too to reset it.~ Ah, it's not persisted to disk if flush isn't called. Nice break.

(this is one of the things I don't like about env var functionality - it's really tricky when it comes to things like this)

Reply:
Yeah I didn't realize the flush disk thing at first either, had a bunch of code to side-step mutating the config.

Was great.... ....

Anyway, added some beforeEach / afterEach to reset the env variable.